src/test/java/bogdanSandu/tests/Tema5/DemoQATests.java

```diff
@@ -0,0 +1,89 @@
+package bogdanSandu.tests.Tema5;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class DemoQATests {
+    private static ChromeDriver driver = null;
+    DemoQATests(String url, ChromeDriver driver) {
+        DemoQATests.driver = driver;
+        driver.get(url);
+    }
+    public void selectTextBox() {
+        WebElement textBox = driver.findElement(By.cssSelector("#item-0"));
+        textBox.click();
+    }
+    public void updateTextField() {
+        WebElement fullName = driver.findElement(By.id("userName"));
+        fullName.click();
+        fullName.clear();
+        fullName.sendKeys("Bob ");
+        WebElement email = driver.findElement(By.id("userEmail"));
+        email.click();
+        email.clear();
+        email.sendKeys("adresamail@gmail.com");
+        WebElement currentAddress = driver.findElement(By.id("currentAddress"));
+        currentAddress.click();
+        currentAddress.clear();
+        currentAddress.sendKeys("Str. No.");
+        WebElement permanentAddress = driver.findElement(By.id("permanentAddress"));
+        permanentAddress.click();
```

Comment:
It would be better to scroll to this element first either via Actions class or via javascript. There is an add banner that covers this field and this causes an exception which will lead to run to fail.
![image](https://user-images.githubusercontent.com/18215854/211187245-871ad8e4-1ee9-4551-abd1-e4f30bbe7a3d.png)